### ISSUE TITLE
BYOIPv6

### ISSUE THREAD

**Issue by jslocinski** [ovh/infrastructure-roadmap#153]
Allow customer to bring IPv6 block into OVHcloud ASN, or bring with IPv6 block together with own AS number and allow to use as Additional IPv6.

**Comment by dotxyzcf** [comment]
ill would also beta test, i really need that and it would be just nice!

**Comment by bjarn** [comment]
Also open to beta test :)!

**Comment by kstawinski** [comment]
@jslocinski, could you please give us any update on this feature?

**Comment by BlueWaterCrystal** [comment]
This feature would be really beneficial, especially with IPv4 exhaustion.

**Comment by davidtoribio** [comment]
# These issues should be addressed within the next 9 to 12 months ?????    can I use my own ipv6 in OVH????

**Comment by jslocinski** [comment]
We had to delay BYOIPv6 delivery (i.e. due to backbone infrastructure upgrade dependencies). Product delivery is planned for summer 2026.
If Alpha/Beta is available before, I will post here.

**Comment by warning31** [comment]
Hello,

I would also like to join the beta. Will there be features like splitting IPv4 blocks and assigning MAC addresses? We are using virtualization (VMware ESXi) and would like to use it without vRack.

Best regards

**Comment by FlorentinDUBOIS** [comment]
Interested as well for Clever Cloud, we have some /44 to deploy :)

**Comment by schleyk** [comment]
I'm also very interested!

**Comment by 001101** [comment]
PUSH PUSH PUSH

**Comment by warning31** [comment]
Hello

When will the IPv6 byoIP update be implemented?

Best regards

**Comment by gabemeola** [comment]
@jslocinski Still on track for this summer? Planning whether OVH will be usable for IPv6 infra or should use another provider in the short term. Thanks!

**Comment by RiRe** [comment]
@jslocinski

**Comment by Mafiyux** [comment]
Any update?

**Comment by jslocinski** [comment]
Hello, we're working on this feature and BYOIPv6 should be available before September this year.

**Comment by 001101** [comment]
no AI agents running at OVH?

**Comment by RiRe** [comment]
Any updates?

**Comment by BlakeB415** [comment]
Will this support BYOIPv6 through the existing Additional IPv6 feature (which currently only works with vRack), or will IPv6 prefixes also be assignable to other product lines, such as VPS instances or dedicated servers without vRack?

**Comment by warning31** [comment]
Hello

When will we start using IPv6 with ByOIP? And will we be able to assign MAC addresses like with IPv4?

Best regards

**Comment by mrc4tt** [comment]
As mentioned in the comment on 29 May. It should be available before September this year. So I will think of 31 August.

/Miksen @ Community Mod on OVHcloud Discord.

**Comment by fixundfertigsein** [comment]
> As mentioned in the comment on 29 May. It should be available before September this year. So I will think of 31 August.

I can't imagine this is an official statement from OVH.
Giving people a specific date when nothing has been officially confirmed isn't right - people plan around these things, and "before September" is an estimate, not a commitment. If there's an official source, provide a link to it.

**Comment by joachimtingvold** [comment]
> I can't imagine this is an official statement from OVH. Giving people a specific date when nothing has been officially confirmed isn't right - people plan around these things, and "before September" is an estimate, not a commitment. If there's an official source, provide a link to it.

@Stephane-Bautista closed #456 yesterday, which is at least some kind of "confirmation" that "something" has happened. I've asked for more explicit feedback and details there.

**Comment by warning31** [comment]
Hello

How much longer will we have to wait? Could an authorized person please provide an explanation?

Best regards

**Comment by thieummm** [comment]
Hi,

BYOIPv6 feature should ship in October. Thanks for your patience.

Mathieu, OVHcloud team

**Comment by thieummm** [comment]
> Will this support BYOIPv6 through the existing Additional IPv6 feature (which currently only works with vRack), or will IPv6 prefixes also be assignable to other product lines, such as VPS instances or dedicated servers without vRack?

Yes it will be supported through Additional IPv6, assignable to vRack.

**Comment by warning31** [comment]
Hello

Will I be able to use it without a vrack? I don't use vracks.

Best regards

**Comment by schleyk** [comment]
can we use the IPv6 Subnet for vps without vrack same with ipv4 right now?

**Comment by Seb-Renaud** [comment]
Hello !

Once you bring your own IPs (whether v4 or v6), they will appear among your Additional IPs: you can handle them as you would with IP provided by OVHcloud. Then you can split/aggregate ranges, move between services and so on...

You may use Additional IP independently from vRack with other products such as VPS, PCI instances, baremetal servers.

VPS support both IPv4 and IPv6, you may read following documentation to move forward with the setup: https://docs.ovhcloud.com/en/guides/bare-metal-cloud/virtual-private-servers/configure-ipv6

I hope it could help.

Regards,

Seb

**Comment by warning31** [comment]
Hello

Can we use it for testing right now?

Best regards.
